--- /opt/retropie/supplementary/runcommand/runcommand.sh_old	2016-05-19 16:10:26.216248289 +0000
+++ /opt/retropie/supplementary/runcommand/runcommand.sh	2016-05-19 19:59:16.784426568 +0000
@@ -789,6 +789,8 @@
 
 retroarch_append_config
 
+/opt/DisplayDaemon/client.py "$system" "$rom_bn"
+
 # launch the command - don't redirect stdout for frotz,  when using console output or when not using _SYS_
 # frotz is included in case its emulators.cfg is out of date and missing CON: - can be removed in the future
 if [[ "$emulator" == frotz || "$is_console" -eq 1 || "$is_sys" -eq 0 ]]; then
@@ -799,6 +801,8 @@
     eval $command </dev/tty &>/tmp/runcommand.log
 fi
 
+/opt/DisplayDaemon/client.py "In menu"
+
 clear
 
 # remove tmp folder for unpacked archives if it exists
